cola/sequenceeditor.py

```diff
@@ -818,8 +830,6 @@ def set_remarks(self, remarks):
                 ">&nbsp;{remark} </span>
             """
         label.setText(label_text)
-        self._parent.setItemWidget(self, self.REMARKS_COLUMN, label)
-        self._parent.resizeColumnToContents(self.REMARKS_COLUMN)
```

Comment:
Thanks for the fix. I'll merge this shortly but I'll probably restore the call to `resizeColumnsToContents` here. The purpose of this call is that it ensure that the column shrinks when remarks are removed.